orbit/changes/17362-desktop-version-and-scripts-enabled

```diff
@@ -0,0 +1,3 @@
+In orbit_info table, added the following fields:
+- desktop_version
+- scripts_enabled
```

Comment:
Nit: Currently, fleetd release automation expects the orbit changes file to be one list item ([PR in review](https://github.com/fleetdm/fleet/pull/18134)).